README.md

```diff
@@ -11,7 +11,13 @@ git clone --recursive https://github.com/wsmoses/Polygeist.git
 cd Polygeist
 ```
 
-### 1. Install LLVM, MLIR, and Clang
+### 1. Install LLVM, MLIR, Clang, and Polygeist.
```

Comment:
To be consistent with the rest can you please remove (or add in the other sentences) the full stop?